openjdk/build.gradle

```diff
@@ -4,7 +4,6 @@ description = 'Conscrypt: OpenJdk'
 
 ext {
     jniSourceDir = "$rootDir/common/src/jni"
-    jniLibPrefix = "conscrypt_openjdk_jni-${osdetector.os}"
```

Comment:
Do we even need `osdetector` then? I'm not sure the `jar.classifier` makes sense anymore.